client/components/Logger.tsx

```diff
@@ -54,6 +54,7 @@ function Loger() {
             userId: '6286978e12716d47e6884194',
         },
     });
+    // eslint-disable-next-line no-console
```

Comment:
Let's leave the warnings in so that we know to come back and remove the console logs. These are just warnings so should be fine to leave in. If it's causing issues we can discuss